app/ante/handler_options.go

```diff
@@ -25,12 +26,14 @@ type HandlerOptions struct {
 	BankKeeper      evmtypes.BankKeeper
 	IBCKeeper       *ibckeeper.Keeper
 	FeeMarketKeeper evmtypes.FeeMarketKeeper
+	SlashingKeeper  *slashingkeeper.Keeper
```

Comment:
This is what `NewParamChangeLimitDecorator` needs in liquidstaking(WIP) and I don't think I need it now

Reply:
I've removed it. https://github.com/b-harvest/Canto/pull/23/commits/9c100e5412c6982c89f2a662ab2c07d8d615ddfd

Reply:
It's removed (https://github.com/b-harvest/Canto/pull/23/commits/a719d632c2f3b5f0afb2378df07fe4b9c3b81bfd)